GameServerConsole/Settings/GameInfo.json.template

```diff
@@ -54,6 +54,6 @@
         "COOLDOWNS_ENABLED": false,
         "CHEATS_ENABLED": true,
         "MINION_SPAWNS_ENABLED": false,
-        "CONTENT_PATH": "../../Content"
+        "CONTENT_PATH": "../../.."
```

Comment:
I think we should still retain an actual content folder, because we will have more than one content pack (like `LeagueSandbox-Default`) in use at the same time. Otherwise the content packs will be scattered in the root of the project

Reply:
The `Content` folder itself can be moved one step upwards though